events/generator/generator.go

```diff
@@ -0,0 +1,42 @@
+package generator
+
+import (
+	"chainguard.dev/sdk/civil"
+)
+
+const GenerateVulnReportEventType = "dev.chainguard.vulnreport.generate.v1"
+
+type Error struct {
+	Status  int    `json:"status"`
+	Code    string `json:"code"`
+	Message string `json:"message"`
+}
```

Comment:
Should we use https://github.com/googleapis/googleapis/blob/2610c7a8f3c362f1ef54cb0bb636ded47422eec8/google/rpc/status.proto#L35 / https://pkg.go.dev/google.golang.org/grpc/status#Status instead? 🤔

Reply:
I changed the code now to use this type.